source/common/common/utility.h

```diff
@@ -657,10 +714,10 @@ template <class Value> struct TrieLookupTable {
   bool add(absl::string_view key, Value value, bool overwrite_existing = true) {
     TrieEntry<Value>* current = &root_;
     for (uint8_t c : key) {
-      if (!current->entries_[c]) {
-        current->entries_[c] = std::make_unique<TrieEntry<Value>>();
+      if (!(*current)[c]) {
```

Comment:
nit: can we use `!= nullptr` here? I feel like that might make it clearer to the reader that `current` holds an array of pointers rather than an array of bools.

Reply:
Done. (Though this complaint belonged on the *previous* iteration of this code, since using `!` on a pointer wasn't a change, and should also be a linter if we want to insist on this.)